pebblo/reports/templates/weasyprintTemplate.html

```diff
@@ -273,44 +303,77 @@
           {% endfor %}
         </tbody>
       </table>
+      {% else %}
+      <div class="w-100 text-center mt-3 bg-surface-50 inter py-8">
+        <img
+          class="mt-1 mb-1 img-datastores"
+          src="../assets/no-data-stores.png"
+          alt="CSV logo"
+        />
+        <div class="w-100 font-13 surface-10">
+          We scanned all your applications and didn’t discover any data source
+        </div>
+      </div>
+      {% endif %}
     </section>
     <section>
       <div>
         <div class="inter flex items-center">
           <div class="font-14 surface-10">Snippets</div>
+          {% if datastores and datastores.findingsDetails and
+          datastores.findingsDetails|length > 0 %}
           <div class="font-12 surface-10-opacity ml-2">
             Showing {{ datastores.displayedSnippetCount }} out of {{
             datastores.totalSnippetCount }}
           </div>
+          {% endif %}
         </div>
-        {% for item in findingDetails %}
-        <div class="mt-4">
-          <div class="flex bg-main pt-2 pb-2 pl-3 pr-3 inter items-center">
-            <div class="surface-10 font-13">{{ item.labelName }}</div>
-            <div class="sub-heading ml-2 surface-50 font-12">
-              Showing {{ item.snippets|length }} snippets out of {{
-              item.snippetCount }}
+        {% if findingDetails and findingDetails|length > 0 %}
```

Comment:
Lets use same condition that we used for local UI for this empty state.

Reply:
Updated condition